src/modules/dashboard/components/CoinMachine/RemainingDisplayWidgets/RemainingTime.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useMemo } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { useDispatch } from 'redux-react-hook';
+
+import { ActionTypes } from '~redux/index';
+import { Address } from '~types/index';
+import { TimerValue } from '~utils/components';
+import useSplitTime from '~utils/hooks/useSplitTime';
+
+import RemainingDisplayWidget from './RemainingDisplayWidget';
+
+export enum DataDisplayType {
+  Time = 'Time',
+  Tokens = ' Tokens',
+}
+
+type Appearance = {
+  theme?: 'white' | 'danger';
+};
+
+type Props = {
+  colonyAddress: Address;
+  value?: string | number | null;
```

Comment:
Now that time and tokens are separated, I think this prop shouldn't be optional. Hell, it could be simpler and you could remove the `string` also